.gitignore

```diff
@@ -1,3 +1,5 @@
 build/.project
 /.cproject
 /build
+.project
+.settings/
```

Comment:
I think it's better to copy what we do in PMDK - ignore all dot files with few exceptions.

Reply:
Done